test/parallel/test-trace-atomic-deprecation.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { test } = require('node:test');
+
+test('should emit deprecation warning DEP0165', async () => {
+  await common.spawnPromisified(process.execPath, ['--trace-atomics-wait', '-e', '{}'])
+        .then(common.mustCall(({ code, stdout, stderr }) => {
+          assert.strictEqual(code, 0);
+          assert.strictEqual(stdout, '');
+          assert.match(stderr, /\[DEP0165\] DeprecationWarning:/);
```

Comment:
Why use a mix of `async/await` and `.then`? 
```suggestion
  const { code, stdout, stderr } = await common.spawnPromisified(process.execPath, ['--trace-atomics-wait', '-e', ';']);
  assert.strictEqual(code, 0);
  assert.strictEqual(stdout, '');
  assert.match(stderr, /\[DEP0165\] DeprecationWarning:/);
```

Reply:
Copy pasted from another test